django/conf/global_settings.py

```diff
@@ -524,6 +524,7 @@
 CSRF_COOKIE_PATH = '/'
 CSRF_COOKIE_SECURE = False
 CSRF_COOKIE_HTTPONLY = False
+CSRF_USE_SESSIONS = False
```

Comment:
In the DutH sprints, @raphaelm and I considered an option to make `CSRF_USE_SESSIONS` default to "use sessions if sessions are being used in the project". We found it a bit too "magical", so we dropped it.

Since the default project template does use sessions, and is likely to prefer to use sessions for CSRF too, it would be nice to make this default to `True`. Regretfully, this would introduce a backwards-incompatibility with AJAX code.

I suggest, though, that we add `CSRF_USE_SESSIONS = True` to the default new-project template.